.vscode/settings.json

```diff
@@ -20,7 +20,8 @@
     "**/.hg": true,
```

Comment:
nit: changes out of scope, this is one file I would leave out of this PR to reduce the PR size (it can be added in a separate PR if it helps)